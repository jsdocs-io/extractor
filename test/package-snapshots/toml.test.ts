import { expectPackageAPIToMatchSnapshot } from '../helpers/expect-package-api-to-match-snapshot';

describe('toml', () => {
    const name = 'toml';

    it('3.0.0', async () => {
        await expectPackageAPIToMatchSnapshot({ name, version: '3.0.0' });
    });
});
