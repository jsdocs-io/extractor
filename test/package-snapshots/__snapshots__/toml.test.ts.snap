// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`toml 3.0.0 1`] = `
{
  "declarations": {
    "classes": [],
    "enums": [],
    "functions": [],
    "interfaces": [],
    "namespaces": [
      {
        "declarations": {
          "classes": [],
          "enums": [],
          "functions": [
            {
              "docs": [],
              "id": "toml.parse",
              "kind": "FunctionDeclaration",
              "name": "parse",
              "signature": "parse: (input: string) => any;",
              "source": {
                "filename": "index.d.ts",
                "line": 2,
                "unpkgURL": "https://unpkg.com/browse/toml@3.0.0/index.d.ts#L2",
                "url": "https://github.com/BinaryMuse/toml-node/tree/4bad34e1dc81586e799e269e5d2c78662b6657b2/index.d.ts#L2",
              },
              "type": "(input: string) => any",
            },
          ],
          "interfaces": [],
          "namespaces": [],
          "typeAliases": [],
          "variables": [],
        },
        "docs": [],
        "id": "toml",
        "kind": "NamespaceDeclaration",
        "name": "toml",
        "signature": "module 'toml' {}",
        "source": {
          "filename": "index.d.ts",
          "line": 1,
          "unpkgURL": "https://unpkg.com/browse/toml@3.0.0/index.d.ts#L1",
          "url": "https://github.com/BinaryMuse/toml-node/tree/4bad34e1dc81586e799e269e5d2c78662b6657b2/index.d.ts#L1",
        },
      },
    ],
    "typeAliases": [],
    "variables": [],
  },
  "files": [
    {
      "filename": "index.d.ts",
      "isIndexFile": true,
      "unpkgURL": "https://unpkg.com/browse/toml@3.0.0/index.d.ts",
      "url": "https://github.com/BinaryMuse/toml-node/tree/4bad34e1dc81586e799e269e5d2c78662b6657b2/index.d.ts",
    },
  ],
  "overview": undefined,
}
`;
